cbioportal_etl/scripts/diff_studies.py

```diff
@@ -214,28 +233,18 @@ def data_clinical_timeline_from_current(url: str, auth_headers: dict, study_id:
     return current_timeline_data
 
 
-def data_clinical_timeline_file_to_list(file_path: str) -> tuple[str, list]:
-    """
-    Read in update flat file headers and data into list
-    """
-    with open(file_path) as flat_file:
-        head = next(flat_file)
-        data: list = flat_file.readlines()
-    return head, data
-
-
-def output_delta_by_id(diff_id: set, update_list: list, header: str, id_field: str, outfile_name: str, big_head: str | None) -> None:
+def output_file_by_id(select_id: set, update_list: list, header: list[str], id_field: str, outfile_name: str, big_head: list | None) -> None:
```

Comment:
Consider doing a default for `big_head`
```suggestion
def output_file_by_id(select_id: set, update_list: list, header: list[str], id_field: str, outfile_name: str, big_head: list | None = None) -> None:
```

Reply:
...I thought defining it as `list | None` was setting a default of `None`...but I guess in reality it's just saying list or None is a possiblity